apps/guardian-ui/src/languages/en.json

```diff
@@ -227,6 +227,7 @@
     "bitcoin-settings": "Bitcoin settings",
     "block-confirmations": "Additional Block Confirmations",
     "block-confirmations-help": "Fedimint Guardians trail the blockchain tip by several confirmations to avoid block reorganizations.",
+    "block-confirmations-help-mainnet": "Running on Mainnet requires at least 5 additional confirmations.",
```

Comment:
Is what you choose in the admin UI the finality delay or the min confirmations? "Additional confirmations" sounds weird.

Reply:
It's 1 confirmation + whatever you set in the UI that gets set, that's why I say "additional" as in "beyond 1"

Reply:
I guess we could just talk about confirmations and then transform to finality delay? On the server side it made some sense when looking from consensus perspective, but for users this seems confusing.

Reply:
> It's 1 confirmation + whatever you set in the UI that gets set, that's why I say "additional" as in "beyond 1"

Should this comment say more than?

https://github.com/fedimint/ui/blob/41e5b309d888992228d2488e9be0dae83c14623f/apps/guardian-ui/src/components/setup/screens/setConfiguration/SetConfiguration.tsx#L167